src/main/java/racingcar/RacingCarContext.java

```diff
@@ -0,0 +1,37 @@
+package racingcar;
+
+import racingcar.controller.RacingController;
+import racingcar.model.car.CarFactory;
+import racingcar.model.car.formatter.CarPositionFormatter;
+import racingcar.model.car.formatter.CarsPositionFormatter;
+import racingcar.model.car.strategy.MovingStrategy;
+import racingcar.model.car.strategy.RandomMovingStrategy;
+import racingcar.model.track.formatter.TrackStateFormatter;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class RacingCarContext {
+    private static RacingController racingController;
+    private static TrackStateFormatter trackStateFormatter;
+
+    public static RacingController randomRacingController() {
+        if (racingController == null) {
+            MovingStrategy randomMovingStrategy = new RandomMovingStrategy();
+            CarFactory carFactory = new CarFactory(randomMovingStrategy);
+
+            racingController = new RacingController(InputView.getInstance(), OutputView.getInstance(), carFactory);
+        }
+
+        return racingController;
+    }
```

Comment:
- 함수명에 동사가 없습니다!
- == null 조건을 붙여준 이유는 어떤 이유로 붙여주신 걸까요? 아래 상황처럼 수정할 수도 있을 것 같습니다!
```suggestion
    public static RacingController randomRacingController() {
            MovingStrategy randomMovingStrategy = new RandomMovingStrategy();
            CarFactory carFactory = new CarFactory(randomMovingStrategy);

            return new RacingController(InputView.getInstance(), OutputView.getInstance(), carFactory);
    }
```

Reply:
`Instance`를 반환해서 해당 Instance 네임을 사용했는데, 함수에 알맞게 네이밍 해보겠습니다!

`== null 조건을 붙여준 이유는 어떤 이유로 붙여주신 걸까요? 아래 상황처럼 수정할 수도 있을 것 같습니다!`
제 의도는 `RacingCarContext`에서 한 번의 `RacingController` 만 생성될 수 있도록 하려고 했습니다. 근데 다시 살펴보니 현재 클래스에서 싱글톤을 보장해주는게 전혀 의미가 없어보이는군요. 인스턴스를 꺼내오는 역할에 맞게 수정해보겠습니다!